behaviors/TranslateBehavior.php

```diff
@@ -92,7 +92,35 @@ public function saveAttributes($event)
         $owner = $this->owner;
         foreach ($this->translateAttributes as $attribute) {
             if ($owner->isAttributeChanged($attribute)) {
-                Language::saveMessage($owner->attributes[$attribute], $this->category);
+                if (Yii::$app->sourceLanguage !== Yii::$app->language) {
+                    $translateSource = LanguageSource::find()
```

Comment:
The use statement for `LanguageSource` is missing, so class not found error occurs when this code runs.